### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by MariusBluem**
Please have a look :) @jospoortvliet @NinaNC @mathiasconradt

### CODE REVIEW

File: page-buy.php [hunk 1]
```diff
@@ -28,7 +28,7 @@
 </section>
 
 <div class="container">
-	<h2><?php echo $l->t('Nextcloud helps you be successful.');?></h2>
+	<h2><?php echo $l->t('Nextcloud helps you beeing successful.');?></h2>
```
Comment (NinaNC) [inline page-buy.php]: beeing > being

File: page-groupware.php [hunk 2]
```diff
@@ -65,9 +65,9 @@
         <div class="col-md-6 revealOnScroll"><p class="section--paragraph__tittle"><?php echo $l->t('Nextcloud Mail');?></p>
 			<p class="section--paragraph"><?php echo $l->t('Send and receive emails in a breeze');?></p>
 			<p class="section--paragraph"><?php echo $l->t('<ul>
-			<li>Multiple accounts</li>
-			<li>Unified inbox</li>
-			<li>Encrypted mail support</li>
+			<li>Multiple accounts with unified inbox</li>
+			<li>Easy to use folder management</li>
```
Comment (NinaNC) [inline page-groupware.php]: Easy-to-use

File: page-groupware.php [hunk 3]
```diff
@@ -65,9 +65,9 @@
...
+			<li>Supports end-to-end encryption using OpenPGP</li>
```
Comment (NinaNC) [inline page-groupware.php]: using PGP > based on PGP? (suggestion)

File: page-groupware.php [hunk 4]
```diff
@@ -77,11 +77,11 @@
...
         </div>
         <div class="col-md-6 revealOnScroll"><p class="section--paragraph__tittle"><?php echo $l->t('Nextcloud Calendar');?></p>
-			<p class="section--paragraph"><?php echo $l->t('Schedule work and meetings online');?></p>
+			<p class="section--paragraph"><?php echo $l->t('Schedule work and your meetings online');?></p>
 			<p class="section--paragraph"><?php echo $l->t('<ul>
-			<li>Add any WebCal calendar</li>
-			<li>Private and public sharing</li>
 			<li>Create multiple calendars</li>
+            <li>Integrate external calendar resources (WebCal)</li>
+			<li>Share your calendars with your team or with the public</li>
```
Comment (NinaNC) [inline page-groupware.php]: I would say "Share your calendars with your team or make it public"

File: page-groupware.php [hunk 5]
```diff
@@ -93,16 +93,16 @@
...
+			<li>Share your addressbooks with your team</li>
+			<li>Sync your contacts with phones and other devices</li>
 			</ul>
 			');?></p>
         </div>
 	</div>
 	<div class="row">
         <div class="col-sm-8 col-sm-offset-2">
-            <p class="section--paragraph revealOnScroll text-center"><?php echo $l->t('Nextcloud Groupware relies on native IMAP and SMTP capabilities. It integrates into existing CalDAV and CardDAV storage while also providing its own.');?></p>
+            <p class="section--paragraph revealOnScroll text-center"><?php echo $l->t('Nextcloud Groupware relies on native IMAP and SMTP capabilities. It fluently integrates your existing infrastructure while also providing its own.');?></p>
```
Comment (NinaNC) [inline page-groupware.php]: integrates into

**Comment by MariusBluem** [comment]
> For the future: should I edit the file directly or put comments like that? What do you prefer @MariusBluem ?

Since content-wise all your edits are fine - I really have no problem if you are editing something ;)

**Comment by jospoortvliet** [comment]
Now one of you has to fix the things Nina pointed out - please, not everybody at the same time 🤣

**Comment by MariusBluem** [comment]
On it.

File: page-buy.php [hunk 6]
```diff
@@ -28,7 +28,7 @@
 </section>
 
 <div class="container">
-	<h2><?php echo $l->t('Nextcloud helps you be successful.');?></h2>
+	<h2><?php echo $l->t('Nextcloud helps you being successful.');?></h2>
```
Comment (jospoortvliet) [inline page-buy.php]: I'm 95% sure it is "help you be successful", not "being", but now you've confused me and I don't know how to find the Truth. Perhaps @jasonbayton can help?

File: page-groupware.php [hunk 7]
```diff
@@ -77,11 +77,11 @@
 			<img class="img-responsive" src="<?php bloginfo('template_directory'); ?>/assets/img/features/calendar-nw.png" />
         </div>
         <div class="col-md-6 revealOnScroll"><p class="section--paragraph__tittle"><?php echo $l->t('Nextcloud Calendar');?></p>
-			<p class="section--paragraph"><?php echo $l->t('Schedule work and meetings online');?></p>
+			<p class="section--paragraph"><?php echo $l->t('Schedule work and your meetings online');?></p>
```
Comment (jospoortvliet) [inline page-groupware.php]: you schedule meetings and work, no need to add 'your'

File: page-groupware.php [hunk 8]
```diff
@@ -77,11 +77,11 @@
...
 			<p class="section--paragraph"><?php echo $l->t('<ul>
-			<li>Add any WebCal calendar</li>
-			<li>Private and public sharing</li>
 			<li>Create multiple calendars</li>
+            <li>Integrate external calendar resources (WebCal)</li>
+			<li>Share your calendars with your team or make it public</li>
```
Comment (jospoortvliet) [inline page-groupware.php]: make 'them' public, then - you're talking about multiple.

File: page-buy.php (hunk 6 above)
Comment (jasonbayton) [reply page-buy.php]: Helps you be is fine, alternatives:

Helps you to be 
Helps you in being
Helps you achieve success

Comment (jospoortvliet) [reply page-buy.php]: You give us options but no solutions. That is either wisdom or annoying. Or both.

I go for achieve, it sounds fancy.

Comment (jasonbayton) [reply page-buy.php]: You're welcome 😜

**Pull request merged**